openedx/core/djangoapps/course_date_signals/tasks.py

```diff
@@ -36,7 +35,7 @@ def update_assignment_dates_for_course(course_key_str):
         if not staff_user:
             LOGGER.error("No staff user found to update assignment dates for course %s", course_key_str)
             return
-        assignments = get_course_assignments(course_key, staff_user)
+        assignments = get_course_assignments(course_key, staff_user, include_without_due=True)
```

Comment:
Why do you need this?
Based on the code `if (due or relative_weeks_due or include_without_due) and graded:` we will return the correct data in case `relative_weeks_due` is present.

Describe please the intention.